packages/content-conformance/src/__tests__/rules.test.ts

```diff
@@ -32,7 +32,7 @@ describe('loadRule', () => {
 
 describe('loadRules', () => {
   test('loads internal rules', async () => {
-    const rules = await loadRules({ 'content-no-h1': 'error' })
+    const rules = await loadRules({ ['content-no-h1' as string]: 'error' })
```

Comment:
This was to address a weird type error, which was saying 'content-no-h1' didn't satisfy the key type of `Record<string, ...>` 🤷